app/views/shelters/show.html.erb

```diff
@@ -9,13 +10,17 @@
     <p>Zip: <%= @shelter.zip %></p>
     <p><%= link_to("View this shelter's pets", "/shelters/#{@shelter.id}/pets", class: "name") %></p>
   </div>
-  <%= link_to 'New Review', "/shelters/#{@shelter.id}/new_review" %>
   <% @shelter.reviews.each do |review| %>
     <div class="card show" id="review-<%= review.id %>">
       <p>Title: <%= review.title %></p>
       <p>Rating: <%= review.rating %></p>
       <p>Content: <%= review.content %></p>
-      <p>Image: <%= image_tag(review.image) if review.image %></p>
+      <% if review.image %>
```

Comment:
if this image just doesn't exist (like in the examples in the test), then this HTML won't be generated at all. if it does exist but is an empty string (this happens when you fill the form out, the image field can be empty but ends up getting stored as ''), then it still won't display. If it exists and is a non-empty string, it will generate the HTML to show the image.